docs/integrations/destinations/bigquery.md

```diff
@@ -6,47 +6,32 @@ description: >-
 
 # BigQuery
 
-## Overview
+## Features
 
-The Airbyte BigQuery destination allows you to sync data to BigQuery. BigQuery is a serverless, highly scalable, and cost-effective data warehouse offered by Google Cloud Provider.
```

Comment:
I think these overview one-liners are useful to keep

Reply:
The information about what BigQuery is can already be found in the description, so the overview is essentially only saying that it allows you to sync data to BigQuery... which I'm not sure is valuable.

<img width="790" alt="Screen Shot 2021-10-03 at 10 50 57 PM" src="https://user-images.githubusercontent.com/33042053/135800512-fa60d96f-89dc-4c44-9dee-5fcbf9b7554b.png">
n

Reply:
I think that it could be valuable keeping all of that information in the meta description, but I'm mostly concerned about organizing the index on the right and making sure the most important information is seen first.